src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -40,6 +40,8 @@ const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
   const isActive = activePaths && includes(activePaths, path)
   const isRootParent =
     activePaths && activePaths.length > 1 && activePaths[0] === path
+  const isLeafItem = children === undefined || children.length === 0
```

Comment:
(not sure we need to change or fix this in this PR) curious - is it undefined or zero length array after all? this check feels a bit excessive.

Reply:
This isn't currently necessary as no sidebar items return empty arrays (and probably won't as long as it's defined as JSON). On top of that, if the issue ever arises we can control for that kind of thing at build time.
Removing this extra check is a micro-optimization, but a valid one nonetheless.